lua/aerial/backends/treesitter/init.lua

```diff
@@ -48,8 +49,15 @@ M.fetch_symbols_sync = function(bufnr)
   -- It is used to determine node parents for the tree structure.
   local stack = {}
   local ext = extensions[lang]
-  ---@diagnostic disable-next-line: missing-parameter
-  for _, matches, metadata in query:iter_matches(syntax_tree:root(), bufnr) do
+  for _, matches, metadata in
+    query:iter_matches(
+      syntax_tree_node,
+      bufnr,
+      syntax_tree_node:start(),
+      syntax_tree_node:end_(),
```

Comment:
```suggestion
      nil,
      nil,
```

these can be `nil` to just accept the default values

Reply:
Thanks! That makes sense. I will update once I am back home.